3-Authorization-II/1-call-api/AppCreationScripts/sample.json

```diff
@@ -1,83 +1,152 @@
-      "ReplyUrls": "http://localhost:3000/",
-      "RequiredResourcesAccess": [
-        {
-          "Resource": "Microsoft Graph",
-          "DelegatedPermissions": ["User.Read"]
-        },
-        {
-          "Resource": "service",
-          "DelegatedPermissions": [ "access_as_user" ]
-        }
-      ],
-      "ManualSteps": [
-        {
-          "Comment": "Navigate to the Manifest page, find the 'replyUrlsWithType' section and change the type of redirect URI to 'Spa'"
-        }
-      ]
-    }
-  ],
-  "CodeConfiguration": [
-    {
-      "App": "spa",
-      "SettingKind": "Replace",
-      "SettingFile": "\\..\\SPA\\src\\authConfig.js",
-      "Mappings": [
-        {
-          "key": "Enter_the_Application_Id_Here",
-          "value": ".AppId"
-        },
+    "ReadmeScenario": {
+        "IncludeFilePath": "",
+        "Image": "",
+        "AdditionalNotes": ""
+    },
+    "ReadmePrerequirements": {
+        "FreeText": "",
+        "IncludeFilePath": ""
+    },
+    "ReadmeSetup": {
+        "FreeText": "",
+        "IncludeFilePath": "",
+        "UseNewSetup": "1",
+        "CertificateOption": "0",
+        "CreateProjectIncludeFilePath": "",
+        "AppRegistrationIncludeFilePath": "",
+        "RunSampleIncludeFilePath": ""
+    },
+    "ReadmeTroubleshooting": {
+        "IncludeFilePath": ""
+    },
+    "ReadmeNextSteps": {
+        "FreeText": "",
+        "IncludeFilePath": ""
+    },
+    "ReadmeContributing": {
+        "FreeText": "",
+        "IncludeFilePath": ""
+    },
+    "ReadmeHowTheCodeWasCreated": {
+        "IncludeFilePath": ""
+    },
+    "ReadmeAboutTheCode": {
+        "IncludeFilePath": ""
+    },
+    "ReadmeAzureDeploy": {
+        "IncludeFilePath": ""
+    },
+    "ReadmeUsingTheSample": {
+        "IncludeFilePath": ""
+    },
+    "ReadmeLearnMore": {
+        "IncludeFilePath": ""
+    },
+    "AADApps": [
         {
-          "key": "Enter_the_Tenant_Info_Here",
-          "value": "$tenantId"
+            "Id": "service",
+            "Name": "msal-node-api",
+            "Kind": "WebApi",
+            "Audience": "AzureADMyOrg",
+            "Sample": {
+                "SampleSubPath": "3-Authorization-II\\1-call-api\\API",
+                "ProjectDirectory": "\\1-call-api\\API"
+            },
+            "Scopes": [
+                "Todolist.Read",
+                "Todolist.ReadWrite"
+            ],
+            "AppRoles": [
+                {
+                    "Types": [
+                        "Application"
+                    ],
+                    "Name": "Todolist.Read.All",
+                    "Description": "Application can only read Todo list"
+                },
+                {
+                    "Types": [
+                        "Application"
+                    ],
+                    "Name": "Todolist.ReadWrite.All",
+                    "Description": "Application can read and write into Todo list"
```

Comment:
>Application can read and write into Todo list [](http://example.com/codeflow?start=36&length=45)

Allow this application to read and write every users ToDo list items

Reply:
done